bigbluebutton-html5/imports/ui/components/menu/component.jsx

```diff
@@ -0,0 +1,151 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { defineMessages, injectIntl } from "react-intl";
+
+import Menu from "@material-ui/core/Menu";
+import MenuItem from "@material-ui/core/MenuItem";
+import { Divider } from "@material-ui/core";
+
+import Icon from "/imports/ui/components/icon/component";
+import Button from "/imports/ui/components/button/component";
+
+import { styles } from "./styles";
+
+const intlMessages = defineMessages({
+  close: {
+    id: 'app.dropdown.close',
+    description: 'Close button label',
+  },
+});
+
+//Used to switch to mobile view
+const MAX_WIDTH = 640;

```

Comment:
We have a [helper](https://github.com/bigbluebutton/bigbluebutton/blob/develop/bigbluebutton-html5/imports/utils/deviceInfo.js#L8) that could be used here to check for mobile devices instead of checking the window size.

Reply:
Yeah I saw the utility to check for mobile, the problem with using that helper is that it excludes regular browsers that are resized to mobile sizes.

![image](https://user-images.githubusercontent.com/22058534/126402363-79ed423c-cf5a-4f2c-90a2-92d9f31d2e19.png)